### DISCUSSION TITLE
I have a problem with auto dark mode

### DISCUSSION THREAD

**Discussion by Arthur-001** [AutoDarkMode/Windows-Auto-Night-Mode#443] [General]
Hi first and for most I'd like to appreciate your great app. When I set dark mod on this app I don't want it to change my toolbar color to light , I want it be in custom mode as I set in settings or just don't change it. When toolbar becomes light it really bothers my eyes. Thanks for your support King.

**Comment by Armin2208** [comment]
what do you mean with toolbar color? the window title bar?

**Reply by Arthur-001** [reply]
I mean taskbar.

**Reply by Armin2208** [reply]
Possible you want to open Auto Dark Mode, want to navigate to "Apps" and there you can disable System theme switch, with setting it to always dark. Or Accent only, how you like.
